apps/console/src/features/applications/components/forms/advanced-configurations-form.tsx

```diff
@@ -107,13 +129,18 @@ export const AdvancedConfigurationsForm: FunctionComponent<AdvancedConfiguration
             return;
         }
 
+        // if apiBasedAuthentication is disabled, then disable clientAttestation as well.
+        const enableClientAttestation: boolean = values.enableAPIBasedAuthentication
+            ? !!values.enableClientAttestation
+            : false;
```

Comment:
Can't we simplify the logic here as follows :D ?

```suggestion
        const enableClientAttestation: boolean = values.enableAPIBasedAuthentication;
```